### DISCUSSION TITLE
Add GTIN, UPC, EAN or ISBN to the Product CSV Importer and Exporter

### DISCUSSION THREAD

**Discussion by racmanuel** [woocommerce/woocommerce#51120] [General]
Hello! In the new WooCommerce update the GTIN, UPC, EAN or ISBN or _global_unique_id is added, I think it would be useful if users could export and import this field natively in woocommerce, using the CSV Importer and Exporter I have done something similar using the following documentation, https://github.com/woocommerce/woocommerce/wiki/Product-CSV-Importer-&-Exporter#adding-custom-import-columns-developers but I don't know if the _global_unique_id is a custom field of the product? I think this function should be natively integrated into the importer and exporter to allow users without programming knowledge to add this field in bulk.

![image](https://github.com/user-attachments/assets/9ac9ea67-8999-4dbb-be9a-7e1deb00bef1)

**Comment by joyryde456** [comment]
https://github.com/woocommerce/woocommerce/issues/51600
